tests/gui/test_orb.py

```diff
@@ -0,0 +1,208 @@
+""" Tests for the Orb UI window """
+from foundry.gui.orb import Store, State, Actions, Action, RomInterface
+from foundry.smb3parse.util.rom import Rom
+
+
+def test_store_initial_state_is_state():
```

Comment:
Should be extracted into three tests.  state3..1 are independent of one another.  This will provide a more descriptive understanding of what the test is achieving.